tests/test_discovery.py

```diff
@@ -47,7 +47,7 @@ def test_run(self):
 
                 # automatic fields will be only primary keys as 
                 # replication keys are not automatically replicated
```

Comment:
@harshpatel4crest Please update the comment as per code changes

Reply:
Updated the comment